jekyll/_cci2/install-machine-runner-3-on-windows.adoc

```diff
@@ -0,0 +1,94 @@
+
+[#prerequisites]
+== Prerequisites
+
+{% include snippets/runner/machine-runner-prereq.adoc %}
+
+[#self-hosted-runner-terms-agreement]
+== Self-hosted runner terms agreement
+
+{% include snippets/runner/terms.adoc %}
+
+[#create-namespace-and-resource-class]
+== 1. Create namespace and resource class
+
+[.tab.machine-runner.Web_app_installation]
+--
+{% include snippets/runner/install-with-web-app-steps.adoc %}
+--
+[.tab.machine-runner.CLI_installation]
+--
+{% include snippets/runner/install-with-cli-steps.adoc %}
+
+--
+
+[#installation-steps]
+== 2. Machine runner installation on Windows
+
+. Download the https://github.com/CircleCI-Public/runner-installation-files/tree/main/windows-install/circleci-runner[`Install-CircleCIRunner.ps1` script] from GitHub to an easily accessible location.
+
+. Open PowerShell as an administrator and navigate to the directory where you placed the script file.
+
+. Run the following in your PowerShell:
++
+[,powershell]
+----
+Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;
+./Install-CircleCIRunner.ps1
+----
++
+The installation will be output into your PowerShell interface.
+
+. As part of the installation, the configuration file for the machine runner (`runner-agent-config.yaml`) will open in Notepad. Fill in the requested information (see xref:machine-runner-3-configuration-reference.adoc[Machine Runner 3.0 Configuration Reference]). The configuration file is located in the installation directory, `C:\Program Files\CircleCI`, by default.
+
+After your setup completes, the machine runner starts automatically and begins looking for jobs to process.
+
+{% include snippets/machine-runner-example.adoc %}
+
+[#uninstall-reinstall-steps]
+== Uninstall / reinstall steps
+
+Uninstalling machine runners will prepare the system for installation again.
+
+. Download the https://github.com/CircleCI-Public/runner-installation-files/tree/main/windows-install/circleci-runner[`Uninstall-CircleCIRunner.ps1` script] from GitHub to an easily accessible location.
+. Open PowerShell as an administrator and navigate to the directory where you placed the script file.
+
+. Run the following in your PowerShell:
++
+[,powershell]
+----
+./Uninstall-CircleCIRunner.ps1
+----
+
+[#continuous-mode-vs.-single-task-mode-for-windows-self-hosted-runners]
+== Continuous mode vs. single task mode for Windows self-hosted runners
+
+By default, Windows machine runners run in xref:machine-runner-3-configuration-reference.adoc#runner-mode[Machine Runner 3.0 Configuration Reference] in order to ensure high reliability of the underlying technology that the self-hosted runner uses to execute jobs. This is the **recommended mode** for Windows machine runners.
```

Comment:
Should this say that they run in `single-mode`? Rather than [Machine Runner 3.0 Configuration Reference]?

Reply:
🤦 yes, yes it should. I'll get that fixed.

Reply:
I noticed that in the config reference doc it says continuous-mode by default, so is it just for Windows that they run in `single-mode`?

Reply:
Other hosts (linux/mac) may want single-mode for various reasons. But on windows particularly there's some quirks, noted just below in the doc, that make single-mode the better option for it.